test/unit/spec/TimeUtil_spec.js

```diff
@@ -0,0 +1,111 @@
+define(['util/TimeUtil'], function (TimeUtil) {
+
+  describe('util/TimeUtil', function () {
+
+    describe('getTimeInHighestRelevantUnit', function () {
+      it('converts millisends', function () {
+        expect(TimeUtil.getTimeInHighestRelevantUnit(0, 'milliseconds')).toEqual({
+          time: 0,
+          unit: 'MILLISECOND',
+        });
+        expect(TimeUtil.getTimeInHighestRelevantUnit(1, 'milliseconds')).toEqual({
```

Comment:
Even if we decide to not handle the edge case of singular (1 millisecond), I think constant name and value should be the same to be less confusing (i.e.: MILLISECONDS = 'milliseconds');